index.html

```diff
@@ -0,0 +1,1376 @@
+                    </p>
+                </article>
+
+            </section>
+            <!--END OF EDITOR'S SECTION 1-->
+        </div>
+        <!--END OF EDITOR'S SECTION 1 CONTAINER-->
+
+
+        <!--START OF EDITOR'S SECTION 2-->
+        <div class="container-fluid">
+            <section class="row mt-4">
+                <article class="col-md-3">
+                    <div class="wrap-cover">
+                        <div class="wrap">
+                            <a href="#">politics</a>
+                        </div>
+                        <div>
+                            <a href="#">
+                                <img src="assets/editors5.jpg"
+                                    alt="Book Better Than The Movie': Journalists Focus On Mueller Performancec"
+                                    class="img-cancer img-fluid" /></a></div>
+
+                    </div>
+                    <p class="editors-headline section-heading font-weight-bold mg-p mt-2 mb-2">
+                        <a href="#" class="feature-hover">Book Better Than The Movie': Journalists Focus On Mueller
+                            'Performance'</a>
+                    </p>
+                    <p class="editors-subheadline mt-0">
+                        Dozens of journalists invoked a classic snobby line of attempted high-brow crticism, comparing
+                        former special counsel Robert Mueller's "unexciting" Wednesday testimony to a book that's better
+                        than the movie.
+                    </p>
+                </article>
+
+                <article class="col-md-3">
+                    <div class="wrap-cover">
+                        <div class="wrap">
+                            <a href="#">OPINION</a>
+                        </div>
+                        <div>
+                            <a href="#">
+                                <img src="assets/editors6.jpg"
+                                    alt="Book Better Than The Movie': Journalists Focus On Mueller Performancec"
+                                    class="img-cancer img-fluid" /></a></div>
+
+                    </div>
+                    <p class="editors-headline section-heading font-weight-bold mg-p mt-2 mb-2">
+                        <a href="#" class="feature-hover">Astonishing Rate of Post Industrial Global Warming Revealed in
+                            New Study</a>
+                    </p>
+                    <p class="editors-subheadline mt-0">
+                        Earth had not seen carbon dioxide concentrations above current levels for at least two million
+                        years.
+                    </p>
+                </article>
+
+                <article class="col-md-3">
+                    <div class="wrap-cover">
+                        <div class="wrap">
+                            <a href="#">WORLD</a>
+                        </div>
+                        <div>
```

Comment:
The same concent of semantic tags `<section>` and `<article>` also applies in FEATURED SLIDESHOWS section.